download_sdl2_hg.cmake

```diff
@@ -26,11 +27,16 @@ endif()
 ExternalProject_Add(
     SDL2HG
     PREFIX ${CMAKE_BINARY_DIR}/external/SDL2
-    URL https://hg.libsdl.org/SDL/archive/default.tar.bz2
+    # URL https://hg.libsdl.org/SDL/archive/default.tar.bz2
+    # Re-enable when the tarball is symlink-free for better Windows compatibility.
+    # In the meantime, use the auto-tracking SDL2 Git repo:
+    GIT_SHALLOW 1
+    GIT_REPOSITORY https://github.com/spurious/SDL-mirror.git
     CMAKE_ARGS
-        "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
-        "-DCMAKE_INSTALL_PREFIX=${CMAKE_BINARY_DIR}"
+        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
```

Comment:
The reason why those are quoted - ability to have space-contained paths. Removal of quotes breaks the support of them.